.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @vvarg229
```

Comment:
This CODEOWNERS file isn't recognizable by GitHub.

Reply:
Fixed.